src/main/resources/config/application.yml

```diff
@@ -207,6 +207,7 @@ application:
     revalCurrentPmJob: ${APPLICATION_CRON_REVALCURRENTPMJOB:0 39 1 * * *}
     revalCurrentPlacementJob: ${APPLICATION_CRON_REVALCURRENTPLACEMENTJOB:0 49 1 * * *}
     recordResendingJob: ${APPLICATION_CRON_RECORDRESENDINGJOB}
+    postFundingStatusSyncJob: ${APPLICATION_CRON_POSTFUNDING_STATUS_SYNCJOB:0 0 3 * * *}
```

Comment:
I know this is a placeholder that is replaced in target environments but please check this time is ok, with regard given to external jobs (NDW).

Reply:
It will be checked with NDW